<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>regist</class>
 <widget class="QDialog" name="regist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK HK</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>정기 주차 등록</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="plate_txt">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>450</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="plate_label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>차량 번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="start_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>정기주차 신청날짜</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>570</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>정기주차 종료날짜</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="start_date">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>410</width>
     <height>200</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_txt">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_txt">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>570</y>
     <width>170</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="register_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	background-color: rgb(148, 198, 65);
	color: rgb(255, 255, 255);
	border-radius: 5px
}
QPushButton:hover{
	
	background-color: rgb(78, 154, 6);
	border-radius: 5px

}</string>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="end_date">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>410</width>
     <height>200</height>
    </rect>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>570</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	background-color: rgb(148, 198, 65);
	color: rgb(255, 255, 255);
	border-radius: 5px
}
QPushButton:hover{
	
	background-color: rgb(78, 154, 6);
	border-radius: 5px

}</string>
   </property>
   <property name="text">
    <string>나가기</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>핸드폰 뒷자리</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_txt">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>490</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>정기 주차 등록 시 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.주차 가격 10% 할인!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.계산은 마지막날 한번에!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
